### ISSUE TITLE
Evaluation on own datasets 

### ISSUE THREAD

**Issue by Mayankjha1535** [junyanz/pytorch-CycleGAN-and-pix2pix#716]
How to evaluate on our own datasets ?

**Comment by junyanz** [comment]
The evaluation depends on your specific task and application. There is no single metric that works for all the datasets and tasks.

**Comment by Mayankjha1535** [comment]
Actually, I have trained the model on my own datasets and Day to night dataset. I want to evaluate it on Mean IOU, Mean pixel accuracy and Mean class accuracy. So, how to evaluate these on my datasets ?

**Comment by junyanz** [comment]
MeanIOU is only useful for the cityscape dataset and other segmentation datasets. It cannot be used in Day2night dataset. You need to design a new metric for your own dataset.

**Comment by diaosiji** [comment]
> MeanIOU is only useful for the cityscape dataset and other segmentation datasets. It cannot be used in Day2night dataset. You need to design a new metric for your own dataset.

Yes, it depends on your specific task. You can find some paper in the area which your own data maybe related to, then you may find some metric，and could compare with Other one.
